File: Move Zeros/Move zeros.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        x = 0
        for i in range(len(nums)):
            if nums[i] != 0:
                nums[i], nums[x] = nums[x], nums[i]
                i+=1
                x+=1
        return nums
                
            

File: Move Zeros/Move zeros.c
void moveZeroes(int* nums, int numsSize){
    int j = 0;
    int i;

    for (i==0; i<numsSize; i++) {
        if (nums[i] != 0) {
            nums[i], nums[j] = nums[j], nums[i];
            i++;
            j++;
        }
        else {
            i++;
            j++;
        }
    return nums;
    }

}